src/app/digest/page.tsx

```diff
@@ -0,0 +1,105 @@
+import { BookOpen } from '@phosphor-icons/react/dist/ssr'
+
+import { Card } from '@/components/Card'
+import { CardGrid } from '@/components/CardGrid'
+import { CTASection } from '@/components/CTASection'
+import { PageHeader } from '@/components/PageHeader'
+import { PageLayout } from '@/components/PageLayout'
+import { PageSection } from '@/components/PageSection'
+import { StructuredDataScript } from '@/components/StructuredDataScript'
+
+import { buildImageSizeProp } from '@/utils/buildImageSizeProp'
+import { createMetadata } from '@/utils/createMetadata'
+import { createTagsProp } from '@/utils/createTagsProp'
+import { extractDescriptionFromContent } from '@/utils/extractDescriptionFromContent'
+import { getDigestArticlesData } from '@/utils/getDigestArticleData'
+import { sortDigestArticles } from '@/utils/sortDigestArticles'
+
+import { graphicsData } from '@/data/graphicsData'
+
+import { attributes } from '@/content/pages/digest.md'
+
+import { PATHS } from '@/constants/paths'
+import { FILECOIN_FOUNDATION_URLS } from '@/constants/siteMetadata'
+
+import { generateStructuredData } from './utils/generateStructuredData'
+
+const { header, seo } = attributes
+const digests = getDigestArticlesData()
+
+export const metadata = createMetadata({
+  seo,
+  path: PATHS.DIGEST.path,
+  overrideDefaultTitle: true,
+})
+
+export default function Digest() {
+  return (
+    <PageLayout>
+      <StructuredDataScript structuredData={generateStructuredData(seo)} />
+      <PageHeader
+        title={header.title}
+        description={header.description}
+        image={graphicsData.imageFallback}
+      />
+
+      <PageSection
+        kicker="Issue 1 - Sep 2024"
```

Comment:
```suggestion
        kicker="Issue 1 - Sept 2024"
```

Reply:
Actually in the design is `Sep` (we are also using `Sep` for the events)